src/group_theory/submonoid.lean

```diff
@@ -151,7 +201,8 @@ begin
   exact list_prod_mem hl
 end
 
-@[to_additive]
+/-- The product of a finset of elements of a submonoid is an element of the submonoid. -/
```

Comment:
It's not literally "a finset of elements of a submonoid" here, but a product _indexed by_ a `finset`.